src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, String color) {
+        this.side = side;
+        setFigureType("square");
+        setColor(color);
+        calculateArea();
+    }
```

Comment:
See my comments to the `Figure` class, firstly change them, and then you will need to change all the inheritors.